File: backend/models/story.go
package models

type Story struct {
	ID            *int    `json:"id,omitempty"`
	Title         *string `json:"title,omitempty"`
	Description   *string `json:"description,omitempty"`
	Status        *string `json:"stat,omitempty"`
	Time          *int    `json:"time,omitempty"`
	DeveloperName *string `json:"developer_name,omitempty"`
}

type StoryInput struct {
	Title         string `json:"title"`
	Description   string `json:"description"`
	Time          int    `json:"time"`
	DeveloperName string `json:"developer_name,omitempty"`
}

File: frontend/src/app/story.ts
export interface Story {
  id: number;
  title: string;
  description: string;
  stat: string;
  time: number;
  developer_name: string;
}

export interface StoryInput {
  title: string;
  description: string;
  time: number;
  developer_name: string;
}
